### PR TITLE
Completed tasks using JavaScript (5 tasks)

### PR DESCRIPTION
**Pull request by SlobodianykOleksandra**
Homework number 2

### CODE REVIEW

File: .idea/workspace.xml
Comment (SlobodianykOleksandra) [inline .idea/workspace.xml:1]: delete this file

File: js_basics/Task1.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let catWeight = 6
```
Comment (sttaran) [inline js_basics/Task1.js]: use `const` for all variables, you don't change them

File: js_basics/Task2.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import chalk from "chalk";
+
+let dogWeight = 28
```
Comment (sttaran) [inline js_basics/Task2.js]: use `const` for all variables, you don't change them

File: js_basics/Task3.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const boyName = 'Yato'
+const girlName = 'Hiyori'
+
+let helloText = 'Main characters of story are '+boyName+' and '+girlName
```
Comment (sttaran) [inline js_basics/Task3.js]: use `const` for all variables, you don't change them

File: js_basics/Task4.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+let heroAge = 25
```
Comment (sttaran) [inline js_basics/Task4.js]: use `const` for all variables, you don't change them

File: js_basics/Task5.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import chalk from "chalk";
+
+//TASK 5.1
+let radius = 8
```
Comment (sttaran) [inline js_basics/Task5.js]: use `const` for all variables, you don't change them

**Review by sttaran — CHANGES_REQUESTED** [review]

**Review by sttaran — APPROVED** [review]

**Pull request merged**